eslint.config.mjs

```diff
@@ -4,6 +4,7 @@ import tseslint from 'typescript-eslint';
 
 /** @type {import('eslint').Linter.Config[]} */
 export default [
+	{ root: true },
```

Comment:
I don't know this way. Does this work the same as in https://github.com/clowder-cloud/frontend/blob/447784da8f4fecbc8f560c78cd7852bdd3abfb5a/.eslintrc.json#L16 `eslintrc.json`?

Reply:
I was not sure as well. I have checked and it looks like that we need root:true here as well. But I am not sure how can I check that it is working correct to be honest.  When I run dev everything runs.

Reply:
I guess if you don't see any errors then it's working. :)